preprocessing/midi_encoder.py

```diff
@@ -56,14 +55,44 @@ def get_track_from_array(array_track, tick_size=15, instrument=1):
     return mid_track
 
 
-def convert_tensor_to_midi(array_tracks, filename):
+def convert_tensor_to_midi(array_tracks, suffix, instrument = 1):
     mid = MidiFile()
 
+    # get these right at some point. maybe read from a filename where it should be encoded? split by " - "
+    artist = "britney"
+    title = "one more time"
+
+    # append meta track 0
```

Comment:
Looks cool, can you make a function that does this, maybe just called `get_default_meta_track`.

Maybe taking `tempo` and `notated32` as arguments?